ci/code_checks.sh

```diff
@@ -138,6 +138,10 @@ if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
 
     # Individual files
 
+    MSG='Doctests algorithms.py' ; echo $MSG
```

Comment:
can these be combined? e.g. 1 for files, 1 for directories

Reply:
Sure, will open a separate PR for that.